src/main/java/duke/main/Ui.java

```diff
@@ -0,0 +1,73 @@
+package duke.main;
+
+import duke.task.Task;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Ui {
+    private Scanner sc = new Scanner(System.in);
+    // Constants
+    private String DIVIDER = "    ____________________________________________________________\n";
+    private String LOGO = "    (• >       (• >       (• >       (• >       (• >       (• >\n"
+            +  "    /))        /))        /))        /))        /))        /))\n"
+            +  "     ``         ``         ``         ``         ``         ``\n";
+    private String HELP = "    IMPORTANT NOTES:\n"
+            + "        Todo: todo <task>\n"
+            + "        Deadline: deadline <deadline> /by <duedate>\n"
+            + "        Event: event <event> /from <start> /to <end>\n"
+            + "        Datetime format: \"dd/MM/yyyy HH:mm\"\n";
+    private String GREET = "    Hello! I'm Birdy\n"
+            + "    chirp chirp! How can I help?\n";
+    private String PARTING = "    chirp! See you around!\n";
```

Comment:
Would it be better for these constants to be declared static and final as well to make it clearer that they are constants?